src/resources/user_address/spec/user_address.spec.js

```diff
@@ -204,3 +213,267 @@ describe('CREATE Address', () => {
     );
   });
 });
+
+describe('UPDATE Address', () => {
+  it('should fail if user is not authenticated', async () => {
+    addressData = await request(app)
+      .post(`/api/v1/users/${dbUser.id}/address`)
+      .set({ 'x-auth-token': userToken, Accept: 'application/json' })
+      .send({ ...address });
+    const response = await request(app)
+      .patch(
+        `/api/v1/users/${dbUser.id}/address/${addressData.body.userAddress.id}`,
```

Comment:
Put any random id here. The id is not gon be checked at this point so no need stressing if it's correct or not